src/test/java/code401challenges/linkedlist/LinkedListTest.java

```diff
@@ -85,18 +86,36 @@ public void notIncludesTest(){
 //    7.Can properly return a collection of all the values that exist in the linked list
     @Test
     public void testToString() {
-        LinkedList test = new LinkedList();
-        test.prepend(3);
-        test.prepend(5);
-        test.prepend(7);
-        test.prepend(9);
-        test.prepend(10);
         assertEquals("This is all the values in the node from beginning to end\nNode Value = 10\nNode Value = 9\nNode Value = 7\nNode Value = 5\nNode Value = 3",
                 test.toString()
         );
     }
 
 
+//    checks to see if the correct value is being inserted at the end of the list.
+    @Test
+    public void appendTest(){
+//        check to see if the to string method can see the new node at the end of the list\
+        test.append(1);
+        assertEquals("This is all the values in the node from beginning to end\nNode Value = 10\nNode Value = 9\nNode Value = 7\nNode Value = 5\nNode Value = 3\nNode Value = 1",
+                test.toString());

```

Comment:
In the future, it would would be better to avoid using toString() to test a different method like append().  One big reason is because someone may change how toString() is implemented, then all these other tests would fail even though the code for them isn't broken.